src/stores/grid_renderer_store.ts

```diff
@@ -659,7 +659,7 @@ export class GridRenderer extends SpreadsheetStore {
     const chipStyle = this.getters.getDataValidationChipStyle(position);
     const border = this.getters.getCellComputedBorder(position, precomputeZone);
 
-    let style = this.getters.getCellComputedStyle(position);
+    let style = this.getters.getCellComputedStyle(position, precomputeZone);
```

Comment:
`viewport` is actually a zone :wink: 
same for borders
```suggestion
    let style = this.getters.getCellComputedStyle(position, viewport);
```